### ISSUE TITLE
Add support for dmg0 boot rom

### ISSUE THREAD

**Issue by Adamillo** [LIJI32/SameBoy#468]
The title of the issue says it all, or rather, suggestion. Support was added for cgb0 so why not this one?

**Comment by LIJI32** [comment]
I’m rather hesitant on adding support to revisions I don’t physically own. For the CGB, I have all revisions, but for the DMG I only have a DMG-B. The situation between CGB and DMG, however, so I might consider it regardless of not having a physical equivalent I can test on.

**Comment by Adamillo** [comment]
Right, I see. Hopefully it's not too hard to find an early DMG revision

**Comment by CasualPokePlayer** [comment]
The DMG0 bootrom should just work (simply replace the dmg_boot.bin), afaik there shouldn't be any actual differences between the two revisions outside of the bootrom (well, except maybe some obscure APU stuff?).

**Comment by LIJI32** [comment]
It will work, but I'm just not too comfortable adding DMG0 support when we don't know how DMG-A, B and C differ. DMG0 is probably just DMG-A with a different boot ROM, but with zero known information about how differently DMG-A behaves compared to DMG-B, which I currently emulate, it's guaranteed to be inaccurate.
